Tools/pixi-packages/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [[ "${PYTHON_ASAN}" == 1 ]]; then
+    echo "BUILD TYPE: ASAN"
+    BUILD_DIR="../build_asan"
+    CONFIGURE_EXTRA="--with-address-sanitizer"
+    export ASAN_OPTIONS="detect_leaks=0:symbolize=1:strict_init_order=true:allocator_may_return_null=1:use_sigaltstack=0"
```

Comment:
per https://github.com/scipy/scipy/pull/24066#discussion_r2604118318
```suggestion
    export ASAN_OPTIONS="strict_init_order=true"
```